### DISCUSSION TITLE
Do we really need Highlight.js ?

### DISCUSSION THREAD

**Discussion by adityatelange** [adityatelange/hugo-PaperMod#385] [General]
Highlight.js(HLJS) has been a part of this theme for a long time.
 
But now what I have observed is HLJS is more of an unnecessary resource being shipped with the theme by default and is enabled by **default**! 

HLJS doesn't help much and most of the code highlights we see are from HUGO itself. This has an impact on the theme's overall performance is what I see.

Let me know what are your thoughts on this.

**Comment by kdkasad** [comment]
I think HLJS should go. Hugo's default highlighting is fine.

**Reply by danielfdickinson** [reply]
I was thinking of a PR for this before I checked the Discussions!

**Comment by pixelfabs** [comment]
Either make it optional or get rid of it all together.
